AIPs/AIP-004.md

```diff
@@ -7,6 +7,9 @@ discussions-to: https://dao.arrowair.com/t/aip-004-discussion-introduce-time-com
 status: Draft
```

Comment:
I think this needs to be changed as well right?

Reply:
Ah, you are right, but I was going to update all the statuses (from 001 -> 004) with the next PR?

Reply:
Ah, I proposed removing 004 from the next PR since it would already be added with the correct statuses through this one. But it's fine, you can use the next PR for that.